server/flaskr/services/notifications/handlers/meeting_started_notification_handler.py

```
@@ -0,0 +1,57 @@
+        return NotificationJob(
+            -1,
+            orgId,
+            meetingId,
+            NotificationType.MeetingStarted.value,
+            "",
+            datetime.now(),
+            None,
+        )
+
+    def update(self, _: list[NotificationJob]):
+        assert False, "Can't update instant notifications"
```

Ruff (0.15.21)

[warning] Assertion always fails, replace with `pytest.fail()`

(PT015)